src/index.html

```diff
@@ -10,13 +10,100 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="header__logo-link-img"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__list-item">
```

Comment:
```suggestion
          </li>
          
          <li class="header__list-item">
```
The same in all places
<img width="1043" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/1699be71-210e-47b7-a8c7-34f3e6ec61a7">